packages/web/src/utils/route/userRouteParser.test.js

```diff
@@ -12,7 +26,7 @@ describe('parseUserRoute', () => {
   })
 
   it('can decode a hashed user id route', () => {
-    mockDecode.mockReturnValue([11845])
+    // Hashids.mockDecode.mockReturnValue([11845])
 
     const route = '/users/eP9k7'
```

Comment:
This test now does not mock the `hashids` module but it works. But we should def figure out how to mock modules used by common... this is as far as I got for now.